packages/ag-charts-website/src/content/docs/supported-frameworks/index.mdoc

```diff
@@ -23,6 +23,7 @@ There are no version compatibility requirements for AG Charts when using the Jav
 
 | AG Charts Version | Angular Version |
 | ----------------- | --------------- |
+| 12.x              | 17 - 20         |
```

Comment:
Might want to consider dropping Angular 17 support here, as [Angular have dropped support for it](https://angular.dev/reference/releases#actively-supported-versions) (Grid is doing the same)

```suggestion
| 12.x              | 18 - 20         |
```

Same would have to happen on the `package.json`'s

Reply:
Tracking distinctly in https://ag-grid.atlassian.net/browse/AG-15017 - if we patch this PR to `b11.3.x` we don't want to also couple it with that change.